torchquad/utils/set_precision.py

```diff
@@ -0,0 +1,30 @@
+import torch
+import os
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def set_precision(data_type="float"):
+    """This function allows the user to set the default precision. Call before declaring any variables!
+    NB: Remember to initialize CUDA first if GPU compatibility is desired.
+
+    Args:
+        data_type (string, optional): Data type to use, either "float" or "double". Defaults to "float".
+    """
+
+    if torch.cuda.is_initialized():
+        if data_type.lower() == "double":
```

Comment:
I think I would differentiate between "float", "double" and if it is neither throw an error. Currently if I pass "complex" which makes no sense, it will just silently go to float